proxy/envoy/testdata/envoy-v0.json.golden

```diff
@@ -1,7 +1,53 @@
 {
   "listeners": [
     {
-      "address": "tcp://0.0.0.0:1081",
+      "address": "tcp://0.0.0.0:80",
```

Comment:
Maybe adding a README file in the testdata dir, or have an md file in doc? I think this can be done separately from this PR.